formulas/dnsmasq/run

```diff
@@ -16,26 +21,36 @@
 #
 #   wickFormula dnsmasq --start
 #
+#   wickFormula dnsmasq --protect-dns
+
 # Returns nothing
 wickGetOption start start "$@"
+wickGetOption protectDns --protect-dns "$@"
 wickPackage dnsmasq
 wickMakeDir /etc/dnsmasq.d/
 wickSetConfigLine /etc/dnsmasq.conf "conf-dir=/etc/dnsmasq.d"
 wickService enable dnsmasq
 wickService force-state dnsmasq "$start"
+wickInfo "This is protectDns: $protectDns"
```

Comment:
If you must keep this line, use wickDebug not wickInfo.  Same for the one below, but the one below shouldn't be needed because all of the arguments to the formula is already written to console when the formula runs.